cmd/fastly-exporter/main.go

```diff
@@ -0,0 +1,328 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	_ "net/http/pprof"
+	"net/url"
+	"os"
+	"os/signal"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/oklog/run"
+	"github.com/peterbourgon/fastly-exporter/pkg/api"
+	"github.com/peterbourgon/fastly-exporter/pkg/prom"
+	"github.com/peterbourgon/fastly-exporter/pkg/rt"
+	"github.com/peterbourgon/usage"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var programVersion = "dev"
+
+func main() {
+	fs := flag.NewFlagSet("fastly-exporter", flag.ExitOnError)
+	var (
+		token       = fs.String("token", "", "Fastly API token (required; also via FASTLY_API_TOKEN)")
+		addr        = fs.String("endpoint", "http://127.0.0.1:8080/metrics", "Prometheus /metrics endpoint")
+		namespace   = fs.String("namespace", "fastly", "Prometheus namespace")
+		subsystem   = fs.String("subsystem", "rt", "Prometheus subsystem")
+		serviceIDs  = stringslice{}
+		includeStr  = fs.String("name-include-regex", "", "if set, ignore any service whose name doesn't match this regex")
```

Comment:
```suggestion
		includeStr  = fs.String("name-include-regex", "", "if set, only include services whose name matches this regex")
```

if set, only include services whose name matches this regex